definition_of_done.md

```diff
@@ -0,0 +1,38 @@
+ Definition of done
+
+This document clarifies, when an product backlog item is ready. During scrum development process this gives us a checklist,
+what needs to be done.
+The individual points are a basis for acceptance criteria and are shortened or appended by specific measures as needed.
+
```

Comment:
Maybe add a general link to the [developtment_guidlines](https://github.com/PilzDE/.github/blob/master/development_guidelines.md)?

Reply:
Imho the guidelines are relevant for the review, not so much for the DoD.

Reply:
Maybe the other way round: Link from development_guidelines here?